### ISSUE TITLE
Rstudio problem

### ISSUE THREAD

**Issue by Rupa-Dey** [rstudio/rmarkdown#2585]
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  namespace ‘xfun’ 0.44 is being loaded, but >= 0.48 is required

when I run my installed library into the R console it works but RMD file does not support it. it creates same problem after restart the R session. now what can I do?

**Comment by cderv** [comment]
> namespace ‘xfun’ 0.44 is being loaded, but >= 0.48 is required

Somehow this means you are in a situation where one of the package requires xfun 0.48 minimum, but you stil have xfun 0.44 installed and loaded in the session. 

> now what can I do?

Open clean R console - no data loaded in workspace, and update **xfun**
````r
install.packages("xfun")
````
This will get you the latest version in your user library and this should get you unstuck. 


If you are a in a project using **renv** or any other configuration modifying your R library, you need to adapt this accordingly. 

To sum up: You need to install latest xfun in a clean R session.

I say clean because sometimes opening a project in RStudio will load some content and package and you will be block. 

You could leverage **pak** : `pak::pak("xfun")`

> when I run my installed library into the R console it works but RMD file does not support it.

Not sure what that means. R Markdown should support latest **xfun**
